src/main/java/edu/rosehulman/quota/Database.java

```diff
@@ -169,4 +169,4 @@ public boolean updateUserTier(UserTier userTier) throws Exception {
     return DaoManager.createDao(connectionSource, UserTier.class);
   }
 
-}
```

Comment:
Somehow a newline was removed here. If you add it back then this file will be removed from the change set.